internal/hostutil/host_utils.go

```diff
@@ -66,3 +67,7 @@ func ValidateHostname(hostname string) error {
 	}
 	return nil
 }
+
+func IgnitionFileName(host *models.Host) string {
+	return fmt.Sprintf("%s-%s.ign", host.Role, host.ID)
```

Comment:
Why do we need the role here?

Reply:
Ask @eranco74 [ref](https://github.com/openshift/assisted-installer/pull/42#discussion_r491941290) :laughing: 

I guess just because the file is specific to the role (it used to be just master.ign or worker.ign). I don't know if we would ever get into a situation where we needed both, role files for a single host id, but this is just more descriptive at a very small cost.